src/prospectScraper/MockDraftPick.cs

```diff
@@ -93,25 +73,12 @@ public static int ConvertPickToRound(string pick)
             }
             
         }
-        public static int convertPickToPoints(string pick, int round)
+
+        public static int ConvertPickToPoints(string pick, int round)
         {
-            int intpick = 0;
-            var canParse = int.TryParse(pick, out intpick);
+            var canParse = int.TryParse(pick, out int intpick);
             if (canParse)
             {
-                /* 
```

Comment:
This comment is also useful and should be kept.

Reply:
Ah, my bad. I tend to have an issue with comments, so I can be aggressive removing them from codebases. I wonder if there is a way to make this information useful, within the console app, without making it a comment? 🤔